### PR TITLE
bug-fix for bright objects without saturated pixels

### PR DESCRIPTION
**Pull request by jchiang87**

### CODE REVIEW

File: imsim/diffraction_fft.py [hunk 1]
```diff
@@ -160,7 +160,10 @@ def apply_diffraction_psf(
         alpha=rottelpos,
         d_alpha=d_alpha,
     )
-    region_row, region_col = saturated_region(image, brightness_threshold)
+    region_info = saturated_region(image, brightness_threshold)
+    if region_info is None:
+        return
```
Comment (rmjarvis) [inline imsim/diffraction_fft.py:165]: Probably want to add a unit test that hits this line.

**Review by rmjarvis — CHANGES_REQUESTED** [review]

File: tests/test_diffraction_fft.py [hunk 2]
```diff
@@ -445,6 +445,30 @@ def test_fft_diffraction_is_similar_to_raytracing_for_field_rotation():
...
+    brightness_threshold = 100000.0
+    spike_length_cutoff = 4000
+
+    image = np.zeros((100, 100))
+
+    diffraction_fft.apply_diffraction_psf(image, wavelength, rottelpos,
+                                          exptime, latitude, azimuth,
+                                          altitude, brightness_threshold,
+                                          spike_length_cutoff)
+
```
Comment (g-braeunlich) [inline tests/test_diffraction_fft.py:470]:
```suggestion
    np.testing.assert_allclose(image, 0.)
```

**Review by g-braeunlich — CHANGES_REQUESTED** [review]
Thx for the fix / unit test!
Just one small suggestion.

File: tests/test_diffraction_fft.py (hunk 2 above)
Comment (jchiang87) [reply tests/test_diffraction_fft.py:470]: Good idea! Thanks.

Comment (jchiang87) [reply tests/test_diffraction_fft.py:470]: @g-braeunlich I committed your change without using the "Commit suggestion" button, so I guess I still need your approval before merging.  Could you please approve?

**Review by g-braeunlich — APPROVED** [review]
:star:

**Pull request merged**